Helper/CriteriaHelper.php

```diff
@@ -149,4 +151,48 @@ public function generateListBlogPostCriterion( Location $location, array $viewPa
         return new Criterion\LogicalAnd( $criteria );
     }
 
+    public function getSortClauseBySortField( $sortField, $sortOrder = Location::SORT_ORDER_ASC )
```

Comment:
We should rather expose this from LocationService then copy it yet another time *(exists in LocationService, UserService and possible other places)*. What is needed to be able to expose this API on LocationService interface cleanly is to be able to type hint Criteria and SortClauses that are supported by Location Search.